tests/test_dataset.py

```diff
@@ -18,6 +25,8 @@
     DATASET_ID_KEY,
 )
 
+TEST_AUTOTAG_DATASET = "ds_bz43jm2jwm70060b3890"
```

Comment:
maybe include a note detailing which user this dataset belongs to

Reply:
Not sure I want to make that information public?